crates/config/src/inline/natspec.rs

```diff
@@ -28,14 +29,27 @@ impl NatSpec {
     pub fn parse(output: &ProjectCompileOutput, root: &Path) -> Vec<Self> {
         let mut natspecs: Vec<Self> = vec![];
 
+        let solc = SolcParser::new();
+        let solang = SolangParser::new();
         for (id, artifact) in output.artifact_ids() {
-            let Some(ast) = &artifact.ast else { continue };
-            let path = id.source.as_path();
-            let path = path.strip_prefix(root).unwrap_or(path);
-            // id.identifier
+            let abs_path = id.source.as_path();
+            let path = abs_path.strip_prefix(root).unwrap_or(abs_path);
+            // `id.identifier` but with the stripped path.
             let contract = format!("{}:{}", path.display(), id.name);
```

Comment:
id.name can be in the form of "{Counter}.0.8.23", we should probably fix this in compilers, I am doing `id.name.split('.').next().unwrap()` usually